src/command/build.rs

```diff
@@ -62,6 +62,9 @@ pub struct BuildOptions {
     debug: bool,
     // build config from manifest
     // build_config: Option<BuildConfig>,
+
+    #[structopt(long = "out-dir", short = "d", default_value = "pkg")]
+    pub out_dir: String,
```

Comment:
This should be a `PathBuf` and then we should use `structopt(parse(from_os_str))` just like the `path` member up above.